### DISCUSSION TITLE
Stop Making Full Version Number Changes

### DISCUSSION THREAD

**Discussion by Goatspunk** [iNavFlight/inav#8994] [Ideas]
Request:  Stop making brand new inav version numbers when only a few small things are changed.  There's no reason to go to a full new version number without much changing.   It's such a pain in the ass to keep having to flash all new versions since they aren't backward compatible for some reason, but are basically the same GUI and operate mostly the same across all numbers. It's also a pain to pull out aircraft you don't normally fly, then have to figure out what version they are running on then hunt around for the iNav configurator specific to that version.  

All version numbers should follow the standard of X.Y.Z where X is the major version, Y is the minor version, and Z is the patch version.

**Comment by MrD-RC** [comment]
INAV version numbers follow the SemVer standard. The version number is set based on compatibility, not perceived feature changes. This is the best way to do it. It’s based on quantifiable information and not perceived new features. 6.0 was a huge update. Not just small changes.
